File: solutions/javascript/eliuds-eggs/1/eliuds-eggs.js
//Just my Python solution
function toBinary(number){
  let binary = ''
  while (number > 0){
    binary = String(number % 2) + binary
    number = Math.floor(number / 2)
  }
  return binary;
}
export function eggCount(displayValue){
  let count=0
  for (const digit of toBinary(displayValue)){
    if (digit==='1'){count++}
  }
  return count
}

File: solutions/python/eliuds-eggs/1/eliuds_eggs.py
def to_binary(number):
    binary = ""
    while number > 0:
        binary = str(number % 2) + binary
        number = number // 2
    return binary
def egg_count(display_value):
    eggs=0
    egg_positions=to_binary(display_value)
    for item in egg_positions:
        if int(item):
            eggs+=1
    return eggs